src/xcoobee/sdk/Consents.js

```diff
@@ -519,6 +519,35 @@ class Consents {
     }
   }
 
+  /**
+   * "Register consents received outside of XcooBee"

```

Comment:
I don't think we need to the double quotes. Please correct me if there is a reason.